classes/module/Module.php

```diff
@@ -3161,7 +3161,8 @@ public function addOverride($classname)
             $override_dest = _PS_ROOT_DIR_ . DIRECTORY_SEPARATOR . 'override' . DIRECTORY_SEPARATOR . $path;
             $dir_name = dirname($override_dest);
 
-            if (!$orig_path && !is_dir($dir_name)) {
+            // Create directory if not exists
+            if (!is_dir($dir_name)) {
                 @mkdir($dir_name, FileSystem::DEFAULT_MODE_FOLDER);
```

Comment:
```suggestion
                @mkdir($dir_name, FileSystem::DEFAULT_MODE_FOLDER, true);
```

You have to allow recursion otherwise it will not work as expected :)